### DISCUSSION TITLE
Hvem representerer du?

### DISCUSSION THREAD

**Discussion by mrosvik** [digdir/designsystemet#2296] [Patterns for public services]
**Hvordan vise at du representerer deg selv eller andre?**
Det bør være tydelig for brukeren om hen representerer seg selv som privatperson, en virksomhet, frivillig organisasjon, eller hjelper en annen privatperson. Hvordan kan vi gjøre dette skillet tydelig med bruk av visuelle og tekstlige virkemidler?

**Comment by mrosvik** [comment]
20. nov 2024: 

- Digdir, Nav og Skatt sjekker hvem som kan stille inn i en arbeidsgruppe for dette. 
- Mulig oppstart Januar 2025. 
- Avklare i ekstra møte i desember hvem som blir med i denne gruppen og hvem som leder arbeidet. (Marianne kaller inn)

**Reply by mrosvik** [reply]
Oppstart er 29. januar 2025. Nav leder og fasiliterer gruppen.

**Comment by Camulos** [comment]
Vi har tilfeller hvor en representerer en "konto", tenk en virksomhet, person eller annen aktør kan ha en eller flere kontoer, med en eller flere tilknyttede brukere.

**Comment by Camulos** [comment]
Typer representasjon kan bli komplisert, men det kan også hende at det finnes noen likhetstrekk som gjøre det mulig å definere en underliggende modell som lar oss bruke et mønster med ganske lite variasjon?:
- Representerer en juridisk person via en registerført rolle (daglig leder, styreleder, forretningsfører, regnskapsfører, revisor)
- Representerer en person via en innskrenking av personens egen styring av eget liv (vergemål, økonomisk styring e.l.)
- Representasjon av en person som enda ikke er myndig grunnet alder (foresatt, forelder, vergemål)
- Bruker tilknyttet en felles konto (konto med flere brukere) i feks en nettbutikk
- Representasjon av en virksomhet (ofte offentlig) gjennom rolle som saksbehandler eller liknende, for eksempel når en skal uttale seg på vegne av statsforvalter eller en etat som har behandlet en sak
- Rolle tilknyttet en person gjennom helsevesen, feks fastlege, koordinator (under utredning eller behandling) eller liknende

**Comment by AudunFK** [comment]
Jeg fikk inn et spørsmål fra Skatt som jeg håper kan inkluderes i diskusjonen om «Hvem representerer du?».

Hvordan ønsker vi at endring av aktør skal fungere i konteksten av skjemautfylling hos Altinn?
Hvis en person har startet utfyllingen av et skjema og oppdager underveis at vedkommende er representerer feil aktør, hvordan skal aktørbytte da fungere?

Skal det være mulig å bytte aktør?
Hvis ja, skal det opprettes et utkast for den opprinnelige aktøren?

**Comment by mrosvik** [comment]
Status: 
Arbeidet som arbeidsgruppen har gjort så langt ligger foreløpig på en [demo-lenke](https://www-pr-3930.victoriousglacier-eb9399b9.norwayeast.azurecontainerapps.io/no/patterns/representation)

Det gjenstår fortsatt en del arbeid med begreper, innhold og eksempler. En tverretatlig gruppe med innholdsdesignere skal jobbe videre med begrepene utover høsten. Når det er på plass, kan vi også lage eksempler, så da nærmer vi oss kanskje noe som er verdt å publisere.

**Comment by frankdahle** [comment]
Har lest gjennom. Synes den er god og finner ingen faglige feil. Veldig spesifikk på anbefalinger som er bra. Bra med alle punktlistene som er lettleste.
Om jeg skal nevne noe, så er det begrepet "underenhet" som ikke helt sitter for min del. Men kommer ikke på noe vesentlig bedre så da får det stå. Det advares jo også om at ikke alle er familiær med begrepet. "underavdeling" kanskje, men funker neppe i alle kontekster.
En annen bagatellmessig nyanse er at ikke alle tilganger krever spesifikk fullmakt. Noen tilganger følger automatisk med visse roller, feks "Daglig leder". Men ikke en viktig presisering i denne konteksten.
Veldig bra!

**Reply by frankdahle** [reply]
Glemte Min side vs Din side. Jeg ville gått litt lengre og sagt at vi anbefaler "Min side", siden det er (mye) mer vanlig/innarbeidet enn Din side. Det er et godt etablert begrep/egennavn, og jeg synes ikke det er et problem at det står "Følg saken din" når det er intuitivt og ikke til å misforstå. Da er det "systemet" som snakker til deg inne i rommet "Min side". Så lenge man er konsekvent er det ok å bruke "din" inne på "Minside".
